pkgs/by-name/ce/celeste-classic/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenvNoCC
+, fetchzip
+, autoPatchelfHook
+, SDL2
+, practiceMod ? false
+}:
+
+let
+  directory = if practiceMod then "CELESTE*Practice*" else "CELESTE";
+  srcbin = if practiceMod then "celeste_practice_mod" else "celeste";
+  outbin = if practiceMod then "celeste-classic-pm" else "celeste-classic";
+in
+stdenvNoCC.mkDerivation {
+  pname = outbin;
+  version = "1.0";
```

We are told this is:
```suggestion
  version = "unstable-2020-12-08";
```
…by hovering on the "2 years ago" text.